src/System.Private.CoreLib/shared/Interop/Windows/BCrypt/Interop.BCryptGenRandom.cs

```diff
@@ -10,7 +10,7 @@ internal partial class Interop
 {
     internal partial class BCrypt
     {
-        internal static unsafe int BCryptGenRandom(byte* pbBuffer, int count)
+        internal static unsafe NTSTATUS BCryptGenRandom(byte* pbBuffer, int count)
         {
```

Comment:
Nit: This wrapper seems to be adding very little value. I am wondering why not call the BCryptGenRandom directly from the two places it is called from.